web/src/app/views/events-view/events-view.component.html

```diff
@@ -80,19 +80,19 @@ <h4>Adresa bazénu</h4>
         <p> TJ ČZU Praha 6 – Kamýcká 1275, Praha, Suchdol, 165 00</p>
         <h4>Sraz</h4>
         <ul class="list-unstyled">
-          <li>- v 16:30, dole ve stanici metra Dejvická (linka A)</li>
+          <li>- dole ve stanici metra Dejvická (linka A), většinou někdy odpoledne (konkrétní čas u daného termínu v programu)</li>
         </ul>
         <h4>Konec</h4>
         <ul class="list-unstyled">
-          <li>- tamtéž, maximálně hodinu po konci bazénu, většinou jsme zpět kolem 19:30</li>
+          <li>- tamtéž, asi 3 hodiny po srazu (půl hodiny tam, 2 hodiny v bazéně, půl hodiny zpátky)</li>
```

Comment:
Nízké číslovky v běžném textu se mají psát slovy, tedy `cca tři hodiny po srazu`. Jinak tady bych zbytek vynechal určitě. Pro rodiče není důležité vysvětlení, jak jsme se k těm třem hodinám dopočítali.

Reply:
ok